lib/usdUfe/ufe/CMakeLists.txt

```diff
@@ -45,6 +46,13 @@ if(CMAKE_UFE_V3_FEATURES_AVAILABLE)
     )
 endif()
 
+if (UFE_SCENEITEM_HAS_METADATA)
+    target_sources(${PROJECT_NAME}
+        PRIVATE
+			UsdUndoableClearSceneItemMetadataCommand.cpp
+			UsdUndoableSetSceneItemMetadataCommand.cpp
```

Comment:
In UsdUfe all the undoable command files are named "UsdUndoXXXCommand" - can you please rename these to match.

Reply:
There's also UsdUndoableCommand.h .cpp but I'll change the above files

Reply:
That one is the base class. It can stay as it is. Just rename your two files.